src/main/kotlin/kotlinx/kover/gradle/plugin/tasks/KoverHtmlTask.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.plugin.tasks
+
+import kotlinx.kover.gradle.plugin.tools.CoverageTool
+import org.gradle.api.file.*
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.*
+import org.gradle.kotlin.dsl.*
+import javax.inject.*
+
+@CacheableTask
+internal open class KoverHtmlTask @Inject constructor(tool: CoverageTool) : AbstractKoverReportTask(tool) {
```

Comment:
I thought we had ability to disable/enable HTML or XML tasks, but I do not see this option anywhere.

Reply:
The best way to disable report generation is not to call the task :)

If it is necessary that the task is not executed, even if it is explicitly or implicitly called - any task inherits the `enabled` property from the `DefaultTask` class.